test/core/parser/segments_common_test.py

```diff
@@ -18,7 +18,7 @@ def test__parser__core_keyword(raw_seg_list):
     # First make a keyword
     FooKeyword = KeywordSegment.make("foo")
     # Check it looks as expected
-    assert issubclass(FooKeyword, KeywordSegment)
+    assert issubclass(FooKeyword.cls, KeywordSegment)
```

Comment:
In this test case, let's rename `FooKeyword` to `foo_keyword_factory`. That makes it more explicit in what's happening in these tests (and below).

Should probably `assert isinstance(foo_keyword_factory, RawSegmentFactory)` too.